test/bats/configs/vault-pki-secretproviderclass.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: vault-pki
+spec:
+  provider: vault
+  parameters:
+    roleName: "example-role"
+    vaultAddress: http://vault:8200
+    vaultSkipTLSVerify: "true"
```

Comment:
Can this value be omitted if the scheme is `http`? I see this in all the examples but it's not relevant to the non-tls address.

Reply:
Good point, it will currently be an error to omit this value and not have certificates configured, but we can definitely be a bit smarter about not requiring that config for `http` schemes.